modules/jdbc/src/main/java/org/testcontainers/jdbc/ConnectionDelegate.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.jdbc;
+
+import lombok.Data;
+import lombok.experimental.Delegate;
+
+import java.sql.*;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.Executor;
+
+class ConnectionDelegate implements Connection {
+    @Delegate(types=Connection.class)
+    private final Connection delegate;
+
+    public ConnectionDelegate(Connection connection){
```

Comment:
nit: you could add `@Value` on the class and remove the constructor and `private final` from the variable declaration

Reply:
`@Value` will not work - it makes class final which prevents from inheriting (see `ConnectionWrapper`)
But `@AllArgsConstructor` will work instead

Reply:
Right 👍